automation-roles/50-install-cloud-pak/cp4d/cp4d-cluster/templates/setup-instance-topology.j2

```diff
@@ -3,5 +3,7 @@ setup-instance-topology \
     --cpd_operator_ns={{ current_cp4d_cluster.operators_project }} \
     --cpd_instance_ns={{ current_cp4d_cluster.project }} \
     --release={{ current_cp4d_cluster.cp4d_version }} \
+    --case_download=true \
```

Comment:
I think the --case_download and --from_oci should only be used with non-airgapped clusters. setup-instance-topology will not download the images if they are already in the case directory and that is how it should work.